### ISSUE TITLE
Citizen Engagement page: Display a message if missing program area detailed info

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#3757]
### Overview
On the Citizen Engagement webpage [1], we want to display a message on a project's program area card if the project is missing program area detailed information so 
- we make sure the page doesn't break and
- we maintain a professional-looking website.

#### Developer
- [x] Read Citizen Engagement wiki page [2]
- [x] If a project's Markdown file is missing one of the following program area detailed information fields: `problem`, `solution`, `impact`, `sdg`, `card-image-src`, `sdg-image-src`, then change the code in `_includes/citizen-engagement-cards.html` [3] so that the card on the Citizen Engagement webpage should show the project's name and dynamically display the message:
  ```
  We are currently drafting the Problem, Solution and Impact statements for this project.
  ```
  - [x] Make sure the message is displayed for any future projects that are added and are missing program area info.
    - Keep in mind missing program area info can mean `problem`, `solution`, `impact`, `sdg`, `card-image-src`, or `sdg-image-src` fields 
      - a) are **not** in a project's Markdown file at all 
      OR
      - b) the fields exist but are left blank in a project's Markdown file.
  - [x] To style the message, use existing SCSS classes when possible, such as for Typography. See `Design System in Figma` [4] for more info
    - [x] If you need to add SCSS code, add it to `_sass/components/_citizen_engagement.scss` [5]
  - [x] Using developer tools, make sure the message looks good in desktop, tablet, and mobile views
- [ ] For testing your code and getting approval from the Stakeholder Rep,
  - [x] In `adopt-civic-art.md` [6] and `curbmap.md` [7] in the `_projects` directory, you can **temporarily** change `visible: false` to `visible: true` in order to see what the card will look like with the message.
  - [x] Show how the message looks on the card to the Stakeholder Rep and get the Stakeholder Rep's approval. 
    - See the Website Team's project information page [8] for who the Stakeholder Rep is.
  - [x] Before opening a pull request, please make sure to change `visible: true` back to `visible: false` in `adopt-civic-art.md` and `curbmap.md` so that the curbmap and ArtWatcher projects are **not** visible.

#### Merge Team, Technical Lead, or Product Manager
- [x] Once the pull request associated with this issue is merged, go to issue #3756 and check if all the issues listed under the `Dependency` section are closed. If they are all closed, move that issue (#3756) into the `New Issue Approval` column, remove the `Dependency` label on it, and add a `ready for dev lead` label to it.

### Resources/Instructions
- [1] [Citizen Engagement webpage](https://www.hackforla.org/citizen-engagement)
- [2] [Citizen Engagement wiki page](https://github.com/hackforla/website/wiki/Citizen-Engagement-page)
- [3] [_includes/citizen-engagement-cards.html](https://github.com/hackforla/website/blob/gh-pages/_includes/citizen-engagement-cards.html): HTML file that provides the layout/format for each card on the Citizen Engagement page
- [4] [Design System in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3&t=IwD8OB61z2yqo4YR-0)
- [5] [_sass/components/_citizen_engagement.scss](https://github.com/hackforla/website/blob/gh-pages/_sass/components/_citizen_engagement.scss): Current SCSS file for the Citizen Engagement page
- [6] [_projects/adopt-civic-art.md](https://github.com/hackforla/website/blob/gh-pages/_projects/adopt-civic-art.md?plain=1)
- [7] [_projects/curbmap.md](https://github.com/hackforla/website/blob/gh-pages/_projects/curbmap.md?plain=1)
- [8] [Website Team's project information page](https://www.hackforla.org/projects/website)
- Issue #3756 is dependent on this issue being completed

**Comment by JessicaLucindaCheng** [comment]
### Issue writing ETA and availability
- Availability: 6 hours
- ETA: Tue, Jan 3, 2023

### Tasks
- [ ] Update the Citizen Engagement wiki page with relevant info from issue #3328 
- [ ] 
- [ ] Write a Medium issue (add to dependency section in 3328) - Change the logic: if it doesn't have the program area info, show the wording
- [ ] Developers can temporarily change curbmap and ArtWatcher to visible to see what the card will look like but then turn it back off when done testing so the temporary change doesn't end up in their pr

**Comment by JessicaLucindaCheng** [comment]
### Issue writing ETA and availability
- Availability: 6 hours
- ETA: Tue, Jan 3, 2023

### Tasks
- [ ] Update the Citizen Engagement wiki page with relevant info from issue #3328 
- [ ] Add instructions to include
  - [ ] Change the logic: if it doesn't have the program area info, show the wording
  - [ ] Developers can temporarily change curbmap and ArtWatcher to visible to see what the card will look like but then turn it back off when done testing so the temporary change doesn't end up in their pr
- [ ] Add as to dependency section in #3328

**Comment by JessicaLucindaCheng** [comment]
### Issue writing ETA and availability
- Availability: Not sure. Prioritizing working on finishing writing other issues.
- ETA: Sun, Jan 14, 2023

### Tasks
- [x] Write a Medium issue (add to dependency section in 3328) - Change the logic: if it doesn't have the program area info, show the wording: `We are currently drafting the Problem, Solution and Impact statements for this project.`
- [x] Developers can temporarily change curbmap and ArtWatcher to visible to see what the card will look like but then turn it back off when done testing so the temporary change doesn't end up in their pr

**Comment by Jaretzbalba** [comment]
Review availability: Weekdays 10-12pm
ETA: 01/24 5pm

**Comment by Jaretzbalba** [comment]
Progress: "I have the code dynamically showing for all of the criteria except for when the alt field exists but is empty. "
Blockers: "Can't seem to figure out how to use the empty or blank tag in liquid to work."
Availability: "4 hours. I am prepping for a job interview so my time is limited right now."
ETA: "2/10"

**Comment by Jaretzbalba** [comment]
@ExperimentsInHonesty Here are screenshots of how the new message looks in mobile, tablet, and desktop views. Please let me know if there are any changes you'd like. Thank you! 

<details>
<summary>Mobile</summary>

![mobile-citizen-engagement](https://user-images.githubusercontent.com/93153059/218538238-8c024395-ac11-4c02-a621-599f78820b39.png)

</details>

<details>
<summary>Tablet</summary>

![tablet-citizen-engagement](https://user-images.githubusercontent.com/93153059/218538269-c46e666c-b879-4bc2-a276-e63a95c8be47.png)

</details>

<details>
<summary>Desktop</summary>

![desktop-citizen-engagement](https://user-images.githubusercontent.com/93153059/218538319-2abd01e1-90f2-43c2-987e-32d5f9d605dd.png)

</details>

**Comment by ExperimentsInHonesty** [comment]
@Jaretzbalba Please make sure that you use the same code that the projects page uses to keep project cards that we don't want displayed from showing. You will notice that the project cards artwatcher and curbmap both have 
`visible: false`
in the md file.

See https://github.com/hackforla/website/blob/09ca8fe04b54ac97edae61f1b677d5250b39770e/assets/js/current-projects.js#L63

FYI @hackforla/website-merge

**Comment by Jaretzbalba** [comment]
@ExperimentsInHonesty Of course! My screenshots above were just used as an example to show what the page would look like if those cards had `visible: true` as requested in the action items for this issue. Before submitting a PR I will change it back to `visible: false`.

**Comment by ExperimentsInHonesty** [comment]
@Jaretzbalba ok, cool!  it looks good to me!

**Comment by ExperimentsInHonesty** [comment]
@hackforla/website-merge ~We need to test this with Access the data, because they said their info was out of date.  Please make another issue to remove their PSI from their project.md card.~

- Actually, there is already another issue. https://github.com/hackforla/website/issues/3885
